### PR TITLE
Assignment4individual marcos de miguel

### PR DESCRIPTION
**Pull request by branfaraco**
Entregando como .py

### CODE REVIEW

File: Assignment4/marcosdmv-22C031/Task06.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+g.add((ns.JaneSmith, vcard.Family, Literal("Smith")))
+# Visualize the results
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.5: Add UPM as the university where John Smith works**"""
+
+# TO DO
+g.add((ns.UPM, RDF.type, ns.University))
+g.add((ns.JohnSmith, vcard.work, ns.UPM))
```
Comment (dgarijo) [inline Assignment4/marcosdmv-22C031/Task06.py:86]: work is not defined in vcard. Search for a property that is defined in that vocabualry

File: Assignment4/marcosdmv-22C031/Task07.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+g.parse(github_storage+"/rdf/example6.rdf", format="xml")
+
+"""**TASK 7.1: List all subclasses of "LivingThing" with RDFLib and SPARQL**"""
+
+# TO DO
+ns = Namespace("http://somewhere#")
+qres = g.query(
+        """SELECT DISTINCT ?a
+           WHERE {
+              ?a rdfs:subClassOf ns:LivingThing
```
Comment (dgarijo) [inline Assignment4/marcosdmv-22C031/Task07.py:31]: this does not retrieve all subclasses of subclasses of living thing

File: Assignment4/marcosdmv-22C031/Task07.py [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+qres = g.query(
+        """SELECT DISTINCT ?a
+           WHERE
+           {{
+            ?a rdf:type ns:Person
+           }
+           UNION
+           {
+            ?a rdfs:subClassOf ns:Person.
+            ?b rdf:type ?a
```
Comment (dgarijo) [inline Assignment4/marcosdmv-22C031/Task07.py:52]: same as above

File: Assignment4/marcosdmv-22C031/Task07.py [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+           WHERE
+           {{{
+            ?a rdf:type ns:Person
+           }
+           UNION
+           {
+            ?a rdfs:subClassOf ns:Person.
+            ?b rdf:type ?a
+           }}
+           FILTER(?a foaf:knows ns:RockySmith)
```
Comment (dgarijo) [inline Assignment4/marcosdmv-22C031/Task07.py:99]: the exercise asks to return  names, not URIs

File: Assignment4/marcosdmv-22C031/Task07.py [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+           }
+           """,
+         initNs = {"rdfs": RDFS, "ns": ns, "rdf": RDF, "foaf": FOAF})
+
+# Visualize the results
+for r in qres:
+  print(r)
+
+"""**Task 7.5: List the entities who know at least two other entities in the graph**"""
```
Comment (dgarijo) [inline Assignment4/marcosdmv-22C031/Task07.py:108]: missing. 

Missing all exercises in rdflib

**Review by dgarijo — CHANGES_REQUESTED** [review]
The exercises are incomplete

**Comment by anaigmo** [comment]
Entregado sin corregir los fallos, no se admiten más PR

**Pull request merged**